src/common/lang/index.js

```diff
@@ -1,10 +1,8 @@
 import Vue from "vue";
 import VueI18n from "vue-i18n";
 import enLocale from "./en";
-import zhLocale from "./zh";
 import LangStorage from './../../helpers/lang'
 import elementEnLocale from "element-ui/lib/locale/lang/en"; // element-ui lang
-import elementZhLocale from "element-ui/lib/locale/lang/zh-CN"; // element-ui lang
 // 按需引入Element
```

Comment:
I suggest to delete or edit this

Reply:
what kind of editing do you have in mind?
i decided to keep the locale object, since we may need adding new languages in future.

Reply:
I meant the comment with the hieroglyphics